deploy/e2e/role.yaml

```diff
@@ -173,6 +173,7 @@ rules:
   - watch
   - create
   - update
+  - patch
```

Comment:
Why is the patch permission needed now?

Reply:
The e2e test which is extending the CA certificate is performing a patch on a `Certificate` resource.